Armadillo/src/main/java/com/scribd/armadillo/encryption/SecureStorage.kt

```diff
@@ -29,6 +29,7 @@ internal interface SecureStorage {
 internal class ArmadilloSecureStorage @Inject constructor(
     @Named(Constants.DI.STANDARD_STORAGE) private val standardStorage: SharedPreferences,
```

Comment:
Can we do the same thing with this store? We should migrate the encryption keys to also be in secure storage